### PR TITLE
add cbirc repo

### PR DESCRIPTION
**Pull request by JkCheung**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
  /cbirc/
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [x] New Route
- [ ] Documentation
  - [x] CN
  - [ ] EN
- [x] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note
     添加 银保监 资讯订阅源

### CODE REVIEW

File: docs/government.md [hunk 1]
```diff
@@ -318,6 +318,16 @@ pageClass: routes
 
 <Route author="hillerliao" example="/csrc/auditstatus/:apply_id" path="/csrc/auditstatus" :paramsDesc="['事项类别id， `https://neris.csrc.gov.cn/alappl/home/xkDetail` 列表中各地址的 appMatrCde 参数']"/>
 
+## 中国银行保险监督管理委员会
+
+<Route author="JkCheung" example="/cbirc/" path="/cbirc/:category" :paramsDesc="['类目']">
```
Comment (linbuxiao) [inline docs/government.md:323]:
```md
- example="/cbirc/"
+ example="/cbirc/jgdt"
---or
+ example="/cbirc"
```

Maybe that would be clearer?

File: lib/v2/cbirc/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/:category?': ['nczitzk'],
```
Comment (TonyRL) [inline lib/v2/cbirc/maintainer.js:2]:
```suggestion
    '/:category?': ['JkCheung'],
```

File: lib/v2/cbirc/index.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+async function getContent(item) {
+    const response = await got({
+        method: 'get',
+        url: 'http://www.cbirc.gov.cn/cn/static/data/DocInfo/SelectByDocId/data_docId=' + item.docId + '.json',
+    });
+    return response.data.data.docClob;
+}
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category || 'ggtz';
```
Comment (TonyRL) [inline lib/v2/cbirc/index.js:81]:
```suggestion
    const category = ctx.params.category ?? 'ggtz';
```

File: lib/v2/cbirc/index.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    const cat = categories[category];
+
+    // 请求集合
+    const response = await ctx.cache.tryGet(cat.link, async () => {
```
Comment (TonyRL) [inline lib/v2/cbirc/index.js:85]: Take a look at https://github.com/DIYgod/RSSHub/pull/8028
I think you can reduce some overhead

File: lib/v2/cbirc/index.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+            header: {
+                Referer: `http://www.cbirc.gov.cn`,
+            },
+        });
+        return resp.data;
+    });
+
+    // 遍历数据集合
+    const dataLs = await Promise.all(
+        response.data.rows.map(async (item) => {
```
Comment (TonyRL) [inline lib/v2/cbirc/index.js:98]: Take a look at https://github.com/DIYgod/RSSHub/pull/8028
I think you can reduce some overhead

**Review by TonyRL — CHANGES_REQUESTED** [review]

**Comment by TonyRL** [comment]
Use `[x]` to tick the checkbox on new RSS checklist

File: lib/v2/cbirc/index.js (hunk 3 above)
Comment (TonyRL) [reply lib/v2/cbirc/index.js:81]: You can mention the default category in the docs.

**Pull request merged**